src/app/components/client/toolbar/AppPicker.tsx

```diff
@@ -0,0 +1,373 @@
+    gaLabel: "vpn",
+  },
+});
+
+/**
+ * Menu that can be opened to see other relevant products Mozilla has available for people.
+ */
+export const AppPicker = () => {
+  const l10n = useL10n();
+
+  const referringHost =
+    typeof document !== "undefined"
+      ? document.location.host
+      : "monitor.firefox.com";
+  const products = getProducts(referringHost);
+  const linkRefs: Record<
+    keyof typeof products,
+    RefObject<HTMLAnchorElement>
+  > = {
+    relay: useRef<HTMLAnchorElement>(null),
+    pocket: useRef<HTMLAnchorElement>(null),
+    fxDesktop: useRef<HTMLAnchorElement>(null),
+    fxMobile: useRef<HTMLAnchorElement>(null),
+    vpn: useRef<HTMLAnchorElement>(null),
+  };
+  const mozillaLinkRef = useRef<HTMLAnchorElement>(null);
+
+  const onSelect = (itemKey: Key) => {
+    Object.entries(products).forEach(([key, productData]) => {
+      if (itemKey === productData.id) {
+        linkRefs[key as keyof typeof products].current?.click();
+        gaEvent({
+          category: "bento",
+          action: "bento-app-link-click",
+          label: productData.gaLabel,
+        });
+      }
+    });
+    if (itemKey === "mozilla") {
+      mozillaLinkRef.current?.click();
+      gaEvent({
+        category: "bento",
+        action: "bento-app-link-click",
+        label: "Mozilla",
+      });
+    }
+  };
+
+  return (
+    <AppPickerTrigger
+      label={l10n.getString("toolbar-app-picker-trigger-title")}
+      onAction={onSelect}
+      referringHost={referringHost}
+    >
+      <Item
+        key={products.vpn.id}
+        textValue={l10n.getString("toolbar-app-picker-product-vpn")}
+      >
+        <a
+          ref={linkRefs.vpn}
+          href={products.vpn.url}
+          className={`${styles["menuLink"]} ${styles.vpnLink}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Image src={VpnLogo} alt="" width={16} height={16} />
+          {l10n.getString("toolbar-app-picker-product-vpn")}
+        </a>
+      </Item>
+      <Item
+        key={products.relay.id}
+        textValue={l10n.getString("toolbar-app-picker-product-relay")}
+      >
+        <a
+          ref={linkRefs.relay}
+          href={products.relay.url}
+          className={`${styles["menuLink"]} ${styles.relayLink}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Image src={RelayLogo} alt="" width={16} height={16} />
+          {l10n.getString("toolbar-app-picker-product-relay")}
+        </a>
+      </Item>
+      <Item
+        key={products.pocket.id}
+        textValue={l10n.getString("toolbar-app-picker-product-pocket")}
+      >
+        <a
+          ref={linkRefs.pocket}
+          href={products.pocket.url}
+          className={`${styles["menuLink"]} ${styles.pocketLink}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Image src={PocketLogo} alt="" width={16} height={16} />
```

Comment:
Same comment as below, but I think the forced `width`/`height` here is causing some stretching for non-SVG icons (the other being the "Firefox for Mobile" logo).

INPUT:
![](https://raw.githubusercontent.com/mozilla/blurts-server/d6fb6bd14bdce089011594546b05635f687ffcef/src/app/components/client/toolbar/images/pocket.png)

OUTPUT:
![](https://user-images.githubusercontent.com/4251/248320644-baac0126-7af6-429c-8faf-a8128f321fa8.png)

Reply:
I also cant tell if it's because I'm looking at a screenshot of a screenshot of a screenshot, or because of the mashed PNG asset, but the Pocket logo looks blurrrrry. Not sure if we can convert from .PNG to .SVG or if resizing to a square bitmap will suffice. I guess the "fx.png" asset is (a) square, and (b) unblurry looking in the screenshot, so maybe it isn't strictly an .PNG issue and more of a stretching-based-blur.

I don't see a bento on the current Monitor site, but on FxA it looks like we have 6 items, although one is "Firefox Monitor". Not sure if we have a 6th item we can add to make it look balanced to avoid that last row gap, or if it'd be weird to have a grid item for "Firefox Accounts". Looks like Relay has an unbalanced 5 grid: https://relay.firefox.com/. And because VPN is now a Bedrock Production(tm) I don't see a bento on that page/site.

For comparison sake, here's the relay.firefox.com bento menu (in all its unsquish'd glory):
<img width="353" alt="Screenshot 2023-06-23 at 2 41 03 PM" src="https://github.com/mozilla/blurts-server/assets/557895/dd52299c-429f-44d9-9459-d73c5a4a70a9">

Reply:
It's not blurry for me, but I did find an SVG (and square) version of the logo at https://blog.getpocket.com/press/, so I've now used that: acbb6e587